src/mpid/ch4/netmod/ofi/ofi_events.h

```diff
@@ -534,7 +534,7 @@ MPL_STATIC_INLINE_PREFIX int MPIDI_OFI_am_recv_event(struct fi_cq_tagged_entry *
          * Put it in the queue to process it later. */
         MPL_DBG_MSG_FMT(MPIDI_CH4_DBG_GENERAL, TERSE,
                         (MPL_DBG_FDEST,
-                         "Expected seqno=%d but got %d (am_type=%d addr=0x%016lx). "
+                         "Expected seqno=%d but got %d (am_type=%d addr=%" PRIu64 "). "
```

Comment:
It just occurred to me we could use `PRIx64` to maintain the hex output, but I'm not sure it really matters. This looks good to me otherwise.

Reply:
Well, if I'm fixing it, I might as well do it right.  Updated.